packages/shortcuts-extension/tsconfig.json

```
@@ -4,7 +4,7 @@
     "outDir": "lib",
     "rootDir": "src"
   },
-  "include": ["src/*"],
+  "include": ["src/*", "src/*/*"],
```

Small simplification

```suggestion
  "include": ["src/**/*"],
```